server/staff/views.py

```diff
@@ -2745,3 +2745,33 @@ def get_context_data(self, **kwargs):
             kwargs['pager'] = pager
 
         return super(SchoolIncomeRecordView, self).get_context_data(**kwargs)
+
+
+class RegionView(BaseStaffView):
+    """
+    地区view: 地区属性编辑, 城市开通.etc
+    """
+
+    def get(self, request, rid, action=None, *args, **kwargs):
+        region = get_object_or_404(models.Region, pk=rid)
+        if action == 'open':
+            return self._open_city(region)
+
+        html_buf = []
+        html_buf.append(region.name)
+        for wts in region.weekly_time_slots.all():
+            html_buf.append(str(wts))
+        return HttpResponse('<br>'.join(html_buf))
+
+    def _open_city(self, region):
```

Comment:
Could a city be opened multiple times? If that happens, there may be junk data, right?